skeleton-typescript-webpack/package.json

```diff
@@ -113,6 +113,7 @@
     "@types/webpack": "^2.0.0",
     "@types/whatwg-fetch": "^0.0.32",
     "aurelia-protractor-plugin": "^1.0.1",
+    "babel-polyfill": "^6.16.0",
```

Comment:
I am using Node.js version 6.2, but it's probably because Wallaby runs a Phantom browser and afaik Phantom does not support the Promise API (yet), right?
You are so right about the Bluebird implementation of Promise. Updated my commit!